html/modules/custom/reliefweb_rivers/templates/reliefweb-rivers-river-article--topic.html.twig

```diff
@@ -0,0 +1,110 @@
+ *   - id: entity id
+ *   - bundle: entity bundle.
+ *   - tags (optional): list of terms the entity is tagged with
+ *   - various properties like posting date, status etc.
+ * - meta_attributes: meta information attributes.
+ *
+ * @todo add back "image" for headlines.
+ */
+
+#}
+<article{{ attributes
+  .addClass([
+    'rw-river-article',
+    'rw-river-article--' ~ entity.bundle,
+    entity.image ? 'rw-river-article--with-image',
+    entity.summary ? 'rw-river-article--with-summary',
+    entity.preview ? 'rw-river-article--with-preview',
+  ])
+  .setAttribute('data-id', entity.id)
+}}>
+  {# Header with country slug and title. #}
+  <header class="rw-river-article__header">
+    {# Country slug. #}
+    {{ render_var({
+      '#theme': 'reliefweb_entities_entity_country_slug',
+      '#countries': entity.tags.country,
+    }) }}
+
+    {# Title. #}
+    {{ render_var({
+      '#theme': 'reliefweb_rivers_river_article_title',
+      '#level': level,
+      '#langcode': entity.langcode,
+      '#url': entity.url,
+      '#title': entity.title,
+    }) }}
+  </header>
+
+  {# Icon (ex: headline). #}
+  {% if entity.icon is not empty %}
+    {{ render_var({
+      '#theme': 'reliefweb_entities_entity_image',
+      '#style': 'medium',
+      '#image': entity.icon,
+      '#caption': false,
+    }) }}
+  {% endif %}
+
+  {# Article attachment preview and/or summary. #}
+  {% if entity.summary is not empty or entity.preview is not empty %}
+  <div class="rw-river-article__content" lang="{{ entity.langcode }}">
+    {% if entity.preview is not empty %}
+    <img src="{{ entity.preview.url }}" alt="{{ entity.preview.alt ?? '' }}">
+    {% endif %}
+    {% if entity.summary is not empty %}
+    <p>{{ entity.summary }}</p>
+    {% endif %}
+  </div>
+  {% endif %}
+
+  {# Footer with meta information (sources, dates etc.). #}
+  <footer class="rw-river-article__footer">
+    {{ render_var({
+      '#theme': 'reliefweb_entities_entity_meta',
+      '#attributes': meta_attributes,
+      '#meta': {
+        'format': {
+          'type': 'simple',
+          'label': 'Format'|t,
+          'value': entity.format,
+          'label_attributes': create_attribute()
+            .addClass('visually-hidden'),
+          'value_attributes': create_attribute()
+            .addClass('rw-entity-meta__tag-value--format--' ~ entity.format|clean_class),
+        },
+        'source': {
+          'type': 'taglist',
+          'label': entity.tags.source|length > 1 ? 'Sources'|t : 'Source'|t,
+          'value': entity.tags.source,
+          'count': 3,
+          'sort': 'shortname',
+          'display': 'shortname',
+        },
+        'posted': {
+          'type': 'date',
+          'label': 'Posted'|t,
+          'value': entity.posted,
+        },
+        'published': {
+          'type': 'date',
+          'label': 'Originally published'|t,
+          'value': entity.published,
+        },
+      }
+    }) }}
+  </footer>
```

Comment:
This can be removed as there is no meta information for the topics in the landing page.